make_atf.sh

```diff
@@ -62,9 +65,13 @@ while read Torig; do
 	    atf_check save_sysctls ${tcptestsuite_path}/${Torig}.pkt
 	    atf_check -o ignore ${packetdrill_path} \
 	        ${tcptestsuite_path}/${Torig}.pkt
-	    atf_check -o ignore sysctl -f saved_sysctls
+	    atf_check restore_sysctls
+	}
+	${Tunder}_cleanup() {
```

Comment:
This will never get run, because you didn't specify "cleanup" on the atf_test_case line.  Also, there's no reason to restore the sysctls twice.  Doing it in the cleanup is sufficient.  I just verified my previous guess: a failing cleanup will make the test broken.

Reply:
Doh.  Thanks for catching the keyword omission.

So, you ran something under `atf_check` in the cleanup?

Reply:
Yeah.  I just ran `atf_check false`.

Reply:
Excellent.  Thanks for your help and thorough review.